INFO     lingkuan_uat.conftest:conftest.py:60 [2025-08-18 18:14] 用例 test_vps/test_vps_orderclose.py::TestVPSOrderSend3::test_trader_orderSend 使用默认URL登录: https://uat.atcp.top/api
INFO     lingkuan_uat.conftest:conftest.py:82 UAT登录尝试 1/5，MFA验证码: 305583
INFO     lingkuan_uat.conftest:conftest.py:107 UAT登录成功（第1次尝试）
INFO     lingkuan_817.commons.variable_manager:variable_manager.py:105 [2025-08-18 18:14] 运行时变量已保存到: D:\pycharm_test\lingkuan_uat\VAR\runtime_vars_vps.json
INFO     lingkuan_uat.conftest:conftest.py:132 切换到VPS URL: https://39.99.145.155/vps
INFO     lingkuan_817.commons.api_base:api_base.py:135 [2025-08-18 18:14] POST请求（JSON）: /subcontrol/trader/orderSend | 数据: {'symbol': 'XAUUSD', 'placedType': 0, 'remark': 'ces', 'intervalTime': 100, 'type': 0, 'totalNum': '3', 'totalSzie': '1.00', 'startSize': '0.10', 'endSize': '1.00', 'traderId': 6171}
INFO     lingkuan_817.commons.api_base:api_base.py:144 [2025-08-18 18:14] 请求后等待 1 秒
ERROR    lingkuan_817.commons.api_base:api_base.py:350 [2025-08-18 18:14] JSON断言失败: 响应msg字段应为success
URL: https://39.99.145.155/vps/subcontrol/trader/orderSend
JSON路径: $.msg
实际值: 300151下单失败,该账号正在下单中
期望值: success | 路径: $.msg | 响应: {"code":500,"msg":"300151下单失败,该账号正在下单中","data":null}